src/test/java/picard/sam/FilterReadsByFlowCellLocationTest.java

```diff
@@ -0,0 +1,163 @@
+package picard.sam;
+
+import htsjdk.samtools.*;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Unit tests for FilterFlowCellEdgeReads using TestNG.
+ */
+public class FilterReadsByFlowCellLocationTest {
```

Comment:
We typically create all our files within the test method and don't use setup-tear down.  Shared fields like this has makes things complicated in ways we would rather avoid.  I believe this is also leaking index files which should ideally be cleaned up.  

These tests just aren't idiosyncratic.  It would probably be better use dataprovider instead of multiple tests like this.